tensorflow/lite/kernels/internal/reference/integer_ops/softmax.h

```diff
@@ -57,11 +52,15 @@ inline void Softmax(const SoftmaxParams& params,
   const int depth =
       MatchingDim(input_shape, trailing_dim, output_shape, trailing_dim);
 
-  int32 min_num = std::numeric_limits<T>::min();
-  int32 max_num = std::numeric_limits<T>::max();
-  OutputParams<T> output_params;
+  int32 min_num = std::is_same<InputT, int16_t>::value
+                      ? 0
+                      : std::numeric_limits<OutputT>::min();
+  int32 max_num = std::numeric_limits<OutputT>::max();
+  int output_num_bits =
+      std::is_same<InputT, int16_t>::value ? 15 : (sizeof(OutputT) * 8);
```

Comment:
```suggestion
      std::is_same<OutputT, int16_t>::value ? 15 : (sizeof(OutputT) * 8);
```

Reply:
same reason as above

Reply:
thanks for review and discussion.

line 63 should be: InputT max_in_row = std::numeric_limits<InputT>::min();
10:45
Current code is a bit misleading, I'll add some comments.